torch/csrc/stable/ops.h

```diff
@@ -217,4 +217,13 @@ inline Tensor copy_(
   return to<Tensor>(stack[0]);
 }
 
+// We expect this to be the stable version of the clone op that takes
+// in no kwargs (memory_format). We will add kwargs support in the
+// future.
+inline Tensor clone(const Tensor& self) {
+  AtenTensorHandle ret = nullptr;
+  TORCH_ERROR_CODE_CHECK(aoti_torch_clone(self.get(), &ret));
```

Comment:
We'd eventually rewrite this with aoti_torch_call_dispatcher (when we add MemoryFormat support) so I'd prefer if this PR did that now (and just passed in an empty value for the third arg).

Reply:
It looks like MemoryFormat is already supported in stable, so I added the optional `memory_format` argument to clone.

Reply:
Now using aoti_torch_call_dispatcher and removed memory_format support.